packages/esl-website/src/navigation/sidebar/sidebar.less

```diff
@@ -69,6 +69,14 @@ esl-d-sidebar {
       display: none;
     }
   }
+
+  .sidebar-hr {
+    display: list-item;
+    border: none;
+    height: 1px;
+    width: 100%;
+    background: linear-gradient(to right, rgb(255, 255, 255), rgba(56, 56, 56, 0));
```

Comment:
Just to my taste, white is a bit too bright, maybe start from .5 opacity of white or some shade of gray.